src/crimson/os/seastore/linked_tree_node.h

```diff
@@ -6,13 +6,10 @@
 #include "crimson/os/seastore/cached_extent.h"
 #include "crimson/os/seastore/transaction.h"
 #include "crimson/os/seastore/root_block.h"
+#include "crimson/os/seastore/btree/btree_types.h"
```

Comment:
I think `btree_types.h` should only be used by fixed-kv-btree (lba and backref) and mappings, currently.

`linked_tree_node.h` is reused by the omap tree, which is irrelavent to the `btree` directory.

So, is it better to move the definition of `btreenode_pos_t` to `seastore_types.h` instead?

Reply:
Looks reasonable to me.